packages/cli/src/templates/expo.ts

```diff
@@ -0,0 +1,277 @@
+    console.log();
+    console.log(`cd ${options.name}`);
+    console.log(`npx expo start`);
+    console.log();
+  },
+};
+
+const APP_TSX_CONTENT = `import { StatusBar } from "expo-status-bar";
+import { StyleSheet, View } from "react-native";
+import { FetchFhirClient } from "@bonfhir/core/r4b";
+import { FhirQueryProvider } from "@bonfhir/query/r4b";
+import { GluestackUIRenderer } from "@bonfhir/gluestack-ui/r4b";
+import { FhirUIProvider } from "@bonfhir/react/r4b";
+import { Text } from "@gluestack-ui/themed";
+
+import PatientComponent from "./PatientComponent";
+
+// Note: if you get [TypeError: Network request failed] errors, you may need to
+// run the following command: ipconfig getifaddr en0
+// replace localhost with the IP address that command returns.
+// related to this issue in react native: https://github.com/facebook/react-native/issues/32931
+const client = new FetchFhirClient({
+  baseUrl: "http://localhost:8103/fhir/R4/",
+  fetch: fetch,
+  auth: {
+    tokenUrl: "http://localhost:8103/oauth2/token",
+    clientId: "f54370de-eaf3-4d81-a17e-24860f667912",
+    clientSecret:
+      "75d8e7d06bf9283926c51d5f461295ccf0b69128e983b6ecdd5a9c07506895de",
+  },
+});
+
+export default function App() {
+  return (
+    <FhirQueryProvider fhirClient={client}>
+      <FhirUIProvider renderer={GluestackUIRenderer}>
+        <View style={styles.container}>
+          <PatientComponent />
+          <Text>This is a FhirValue Component ^</Text>
+          <StatusBar style="auto" />
+        </View>
+      </FhirUIProvider>
+    </FhirQueryProvider>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+});
+`;
+
+const PATIENT_COMPONENT_CONTENT = `import { FhirValue } from "@bonfhir/react/r4b";
+import { useFhirRead } from "@bonfhir/query/r4b";
+import { Text } from "@gluestack-ui/themed";
+
+export default function PatientComponent() {
+  const patientData = useFhirRead(
+    "Patient",
+    "479cb82c-221d-480d-a6a5-e3eedd47279c"
+  );
+
+  if (patientData.isLoading) {
+    return <Text>Loading...</Text>;
+  }
+
+  if (patientData.error) {
+    return <Text>{JSON.stringify(patientData, null, 2)}</Text>;
+  }
+
+  return <FhirValue type="HumanName" value={patientData.data?.name} />;
+}
+`;
+
+const TSCONFIG_CONTENT = `{
+    "extends": "expo/tsconfig.base",
+    "compilerOptions": {
+      "moduleResolution": "bundler",
+      "strict": true
+    }
+  }
+  `;
+
+const METRO_CONFIG_CONTENT = `// Learn more https://docs.expo.io/guides/customizing-metro
+const { getDefaultConfig } = require('expo/metro-config');
+
+/** @type {import('expo/metro-config').MetroConfig} */
+const config = getDefaultConfig(__dirname);
+
+// Enable Package Exports
+config.resolver = {
+  ...config.resolver,
+  unstable_enablePackageExports: true,
+};
+
+config.resolver.sourceExts.push('mjs');
```

Comment:
This could probably be written as a single instruction:


```suggestion
config.resolver = {
  ...config.resolver,
  sourceExts: [
    ...config.resolver.sourceExts,
    "mjs"
  ],
  unstable_enablePackageExports: true,
};
```